static/src/javascripts/projects/common/modules/commercial/user-features.spec.js

```diff
@@ -296,10 +296,10 @@ describe('The isDigitalSubscriber getter', () => {
             expect(isDigitalSubscriber()).toBe(false);
         });
 
-        it('Is true when the user has no digital subscriber cookie', () => {
+        it('Is false when the user has no digital subscriber cookie', () => {
             // If we don't know, we err on the side of caution, rather than annoy paying users
```

Comment:
Now we err on the side of annoying paying users

Reply:
Ack, I mean: we can remove this comment because now you're introduced more sensible logic. 😅